.github/workflows/integration.yaml

```diff
@@ -62,15 +67,35 @@ jobs:
         with:
           apikey: ${{ secrets.BUILDEVENTS_APIKEY }}
           dataset: gha-buildevents_integration
-          job-status: ${{ job.status }}
           matrix-key: ${{ matrix.value }}
 
       - run: |
-          echo "STEP_ID=0" >> $GITHUB_ENV
+          STEP_ID=matrix-${{ matrix.value }}
+          echo "STEP_ID=${STEP_ID}" >> $GITHUB_ENV
           echo "STEP_START=$(date +%s)" >> $GITHUB_ENV
       - run: |
           buildevents cmd $TRACE_ID $STEP_ID ${{ matrix.value }} -- echo The matrix value is ${{ matrix.value }}
       - run: |
           buildevents cmd $TRACE_ID $STEP_ID sleep -- sleep 2
       - run: |
-          buildevents step $TRACE_ID $STEP_ID $STEP_START 'step 1'
+          buildevents step $TRACE_ID $STEP_ID $STEP_START $STEP_ID
+
+  end-trace:
+    runs-on: ubuntu-latest
+    needs: [smoke-test, matrix]
+    if: ${{ always() }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: technote-space/workflow-conclusion-action@v3
+      - name: "Honeycomb: install buildevents"
```

Comment:
Just noticed this: is this name correct here? We're ending the trace so it seems weird to say "install buildevents"?

Reply:
oh that is the step name that is setting up the buildevents cli that gets called (the entire job is called end trace I guess)

Reply:
Ah, okay! Thanks for explaining